ethcore/src/block_queue.rs

```diff
@@ -38,6 +38,8 @@ pub struct BlockQueueInfo {
 	pub verified_queue_size: usize,
 	/// Number of blocks being verified
 	pub verifying_queue_size: usize,
+	/// Indicates queue is empty
+	pub empty: bool
```

Comment:
function for BlockQueueInfo?
seems reasonable
I guess 'full' should be function as well then, right?